plugins/User/config.py

```diff
@@ -43,8 +43,9 @@ def configure(advanced):
 
 
 User = conf.registerPlugin('User')
-# This is where your configuration variables (if any) should go.  For example:
-# conf.registerGlobalValue(User, 'someConfigVariableName',
-#     registry.Boolean(False, """Help for someConfigVariableName."""))
+conf.registerGlobalValue(User, 'listInPrivate',
+    registry.Boolean(True, _("""Determines whether the output of 'user list'
+    will be sent in private. This prevents mass-highlights of people who use
+    their nick as their bot username.""")))
```

Comment:
The config value appears to override supybot.reply.inPrivate. 
https://github.com/ProgVal/Limnoria/blob/testing/src/callbacks.py#L174-L175 - supybot.reply.inPrivate only takes effect if the private argument is None, and is ignored if private is True or False.